tests/OSPSuite.Starter/Services/ParameterToParameterDTOMapper.cs

```diff
@@ -22,6 +23,28 @@ public class ParameterDTO : ValidatableDTO<IParameter>, IParameterDTO
       public ParameterDTO(IParameter parameter) : base(parameter)
       {
          Parameter = parameter;
+         bindToParameter();
+      }
+
+      private void bindToParameter()
```

Comment:
This code is in the test program to ensure that I have an update in the view (was not working before that). Again object is in test program so no problemo